src/Application/StatementListTranslator.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace ProfessionalWiki\WikibaseFacetedSearch\Application;
+
+use Wikibase\DataModel\Entity\ItemId;
+use Wikibase\DataModel\Entity\PropertyId;
+use Wikibase\DataModel\Statement\StatementList;
+
+class StatementListTranslator {
+
+	public function __construct(
+		private readonly StatementTranslator $statementTranslator,
+		private readonly InstanceTypeExtractor $instanceTypeExtractor,
+		private readonly Config $config
+	) {
+	}
+
+	/**
+	 * @return array<string, mixed>
+	 */
+	public function translateStatements( StatementList $statements ): array {
```

Comment:
This could use a short comment about what the statements are being translated to

Reply:
https://github.com/ProfessionalWiki/WikibaseFacetedSearch/pull/62